src/storage/statistics_cl.c

```diff
@@ -542,7 +534,7 @@ stats_ndv_dump (const char *class_name_p, FILE * file_p)
     }
 
   /* get NDV by query */
-  if (stats_get_ndv_by_query (class_mop, &class_attr_ndv, file_p) != NO_ERROR)
+  if (stats_get_ndv_by_query (class_mop, &class_attr_ndv, file_p, 0) != NO_ERROR)
```

Comment:
This is the magic number. I think we should use the following.
```suggestion
  if (stats_get_ndv_by_query (class_mop, &class_attr_ndv, file_p, STATS_WITH_SAMPLING) != NO_ERROR)
```

Reply:
I fixed it.